main.ts

```diff
@@ -493,6 +493,12 @@ function renderAcademics(options) {
       return ((d) * 10) + '%'
     }
   });
+  renderLineChart(d3.select('#grades-official'), GRADES_OFFICIAL, options.width, 300, {
+    toggle: 'gender-salary',
```

Comment:
curious about this toggle -- what does it do?

Reply:
@jaxonL Oh I think that's from me actually. I made some change to the line graph so you can have them interactive where needed. The toggle specifies what the class name for each line is for the purpose of toggling between visible and hidden. (for line 1, the class would be `gender-salary-0`)
I think with my change you need to provide a toggle field now, but maybe you don't I can't really remember what I did :/